enterprise/servletjspapi/external/javax.servlet.jsp-api-2.3.3-license.txt

```diff
@@ -0,0 +1,833 @@
+Name: Java API for Servlets, Java API for JavaServer Pages
+Version: 2.3.3
+License: CDDL-GPL-2-CP
+Description: Java API for Servlets, Java API for JavaServer Pages, Java API for Expression Language
+Origin: Oracle (https://glassfish.dev.java.net/)
+OSR: 5563
```

Comment:
We can remove this line.

Reply:
Thanks Junichi. I have removed this line in the latest commit.